packages/bot-web-ui/src/components/dashboard/tutorial-tab/sidebar.tsx

```diff
@@ -34,7 +34,7 @@ const Sidebar = ({
     const [search_faq_list, setsearchFAQList] = React.useState(faq_content);
     const search_input = React.useRef<HTMLInputElement | null>(null);
     React.useEffect(() => {
-        if (search_input?.current?.value) setsearchFAQList((search_input.current.value = ''));
```

Comment:
@maryia-matskevich-deriv i had added this value to clear the line input field.. adding empty array is not clearing the field

Reply:
@rupato-deriv how to reproduce it? I never get console.log checking `search_input?.current?.value`

Reply:
@rupato-deriv type of search_faq_list === type of faq_content. type of faq_content is array of objects. If we want to clean up array of objects - the way to do it is set empty array

Reply:
fixed ✅